src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -162,6 +162,21 @@ public final class InlineConfigParser {
     private static final Pattern VIOLATIONS_SOME_LINES_ABOVE_PATTERN = Pattern
             .compile(".*//\\s*(\\d+) violations (\\d+) lines above:$");
 
+    /**
+     * <p>
+     * Multiple violations for line. Violations are Y lines below, messages are X lines below.
+     * {@code
+     *   // X violations Y lines below:
+     *   //                            'violation message1'
+     *   //                            'violation messageX'
+     * }
+     *
+     * Messages are matched by {@link InlineConfigParser#VIOLATION_MESSAGE_PATTERN}
+     * </p>
+     */
+    private static final Pattern VIOLATIONS_SOME_LINES_BELOW_PATTERN = Pattern
+            .compile(".*//\\s*(\\d+) violations (\\d+) lines below:$");
+
```

Comment:
Added support for new kind of violation msg similar to 

https://github.com/checkstyle/checkstyle/blob/8d5e2e2c212f3b988abd2a03fb980141ecae42e3/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L150-L163

so we can display violations in a more readable way